oioswift/server.py

```diff
@@ -50,10 +50,14 @@ def __init__(self, conf, memcache=None, logger=None, account_ring=None,
                                   container_ring=container_ring)
         if conf is None:
             conf = {}
-        self.sds_namespace = conf.get('sds_namespace', 'OPENIO')
-        self.sds_proxy_url = conf.get('sds_proxy_url', 'http://127.0.0.1:6000')
+        sds_conf = {k[4:]: v
+                    for k, v in conf.iteritems()
+                    if k.startswith("sds_")}
+        self.sds_namespace = sds_conf.pop('namespace', 'OPENIO')
+        self.sds_proxy_url = sds_conf.get('proxy_url', 'http://127.0.0.1:6000')
```

Comment:
This comment doesn't precisely concern the current PR, but it is worth, though.

Default configuration values are only acceptable when it could work whatever the deployment. Namespaces and moreover URLs are a typical counter-example of default-able variables. Especially the URLS that could actually point to a valid service from another namespace.